### DISCUSSION TITLE
Scrolling on mobile phones from center space of the model viewer component actually makes them scroll the page.

### DISCUSSION THREAD

**Discussion by sameer-manek** [google/model-viewer#3081] [Q&A]
So model viewer on phones works great. But on phones I can scroll past the model if I try to turn the model from center. it seems to work okay if I grab the model from either left or right edge and it looks like it is the default behavior. Anyway I can fix that?

**Comment by elalish** [comment]
Our  default  behavior is `touch-action="pan-y"`, which can be  changed to "none" here: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-touch-action

The default takes drags that start vertical and sends them to scroll, while drags  that start horizontal become model rotation.
